tests/support/loadJsdom.ts

```diff
@@ -31,6 +31,13 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 
 global.cancelAnimationFrame = () => {};
 
+global.IntersectionObserver = function () {
```

Comment:
Using `test-extras` for unit testing in widget-core has issues, but we could probably load the Jsdom module. It will mean that we need to have a dev dependency on a released version of widget-core (due to peer deps requirements of test-extras).